KMB/kmb_massload.py

```diff
@@ -169,34 +169,46 @@ def process_byline(entry):
         entry['byline'] = helpers.flip_name(entry['byline'])
 
 
-# @todo: update this per new copyright rules - T164568
 def process_license(entry):
     """
-    Identify the license and store back in entry.
+    Identify the license and store the back in entry.
```

Comment:
That is a left-over from the old function. The "the" was all that got saved of me updating it.
Done